# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcTCPSocketChild.h
# Commit: 8196045d38b5
# Full Hash: 8196045d38b50e04313725215c36ba1618c50486
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Create an ipdl struct called WebrtcProxyConfig, to cut down on the arg count on lots of functions. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45562
# ==============================================================================

diff -r 6e1eba5c88ad -r 8196045d38b5 media/mtransport/ipc/WebrtcTCPSocketChild.h
--- a/media/mtransport/ipc/WebrtcTCPSocketChild.h	Wed Sep 18 14:27:42 2019 +0000
+++ b/media/mtransport/ipc/WebrtcTCPSocketChild.h	Tue Sep 17 18:19:04 2019 +0000
@@ -31,9 +31,8 @@
 
   void AsyncOpen(const nsCString& aHost, const int& aPort,
                  const nsCString& aLocalAddress, const int& aLocalPort,
-                 bool aUseTls, const Maybe<net::LoadInfoArgs>& aArgs,
-                 const Maybe<nsCString>& aAlpn, const Maybe<dom::TabId>& aTabId,
-                 const Maybe<NrSocketProxyConfig::ProxyPolicy>& aProxyPolicy);
+                 bool aUseTls,
+                 const std::shared_ptr<NrSocketProxyConfig>& aProxyConfig);
 
   void AddIPDLReference() { AddRef(); }
   void ReleaseIPDLReference() { Release(); }